pubspec.yaml

```diff
@@ -38,13 +38,14 @@ dependencies:
   rust_builder:
     path: rust_builder
   flutter_rust_bridge: 2.0.0-dev.5
-  intl: ^0.19.0
+  intl: ^0.17.0
   flutter_secure_storage: ^9.0.0
   path_provider: ^2.1.3
   qr_flutter: ^4.1.0
   qr_code_scanner: ^1.0.1
   flutter_svg: ^2.0.10+1
-  share: ^2.0.4
+  share: ^2.0.4 
+  custom_qr_generator: ^0.1.2
```

Comment:
please set the version number of this dependency explicitly so that we do not pick up newer versions when updating dependencies in the future

`custom_qr_generator: 0.1.2`